### DISCUSSION TITLE
Questions on Splunk OpenTelemetry Collector for Kubernetes

### DISCUSSION THREAD

**Discussion by oarusoft** [signalfx/splunk-otel-collector-chart#530] [Q&A]
I have few addtional questions when working on issue [https://github.com/signalfx/splunk-otel-collector-chart/discussions/526](url).

Below are questions 

1.Where can I see all the helm splunkPlatform.XXXXX flags . Is there any refrence documets to see all flags
2. Helm have option to add metricIndex too without token. But in my prod env I have metricsIndex token too. My question is what is the use of metricsIndex token if I am able to use mertic index name directly. Is it so that I can use metrics token directly ?
3.Once I am able to collect OpenTelemetry data for Kubernetes how can I easily create dashboards from data. Is the any splunkbase  install or publically avilable dashboard that I copy from?

Please advice. Thank You.

**Comment by atoulme** [comment]
1. The reference is here: https://github.com/signalfx/splunk-otel-collector-chart/blob/1d7788e9aca544fa86c2ebbf0d1f11c8adbc33a8/helm-charts/splunk-otel-collector/values.yaml#L25
2. I'm not sure, this might require an enhancement to offer this as a separate option. Please open an issue.
3. This depends on how you are set up. You should reach out to your Customer Success Manager for help and look at options with you.

**Reply by oarusoft** [reply]
Thanks @atoulme for answering 1. 
Regarding my question 3, Once I get data into splunk, I am having hard time building metric in splunk. 
Can I get some basic metrics example that i can extract in Splunk? For example can I get pod cpu and memory usage?
Thank you.
